master.js

```diff
@@ -47,7 +48,7 @@ const DEFAULT_APP_FUNCTIONS = ['Master', 'Collector', 'Updater'];
  * In case of health-check succeeds a custom health check function should call callback(null), otherwise return an error object constructed with a help of.
  * errorStatusFmt() function. For example, master.errorStatusFmt('ALAZU00001', 'Some error description');
  * 
- * @param {Array.<Function>} collectionStatsFun - (optional) a function which is called during checking to get collection stats. The result of the function will be assigned to 'collection_stats' property of a checkin body. Default is null.
+ * @param {Array.<Function>} collectionStatsFun - (optional,deprecated) a function which is called during checking to get collection stats. The result of the function will be assigned to 'collection_stats' property of a checkin body. Default is null.
```

Comment:
we can't because it will brake existing collectors right away.
In general it still can be useful from for custom checkin functionality, but is not use for now.